### DISCUSSION TITLE
Help wanted for fixes and developments

### DISCUSSION THREAD

**Discussion by danielbarion** [ReactTooltip/react-tooltip#803] [General]
Hi @aronhelser @alexgurr @pdeszynski, I'm trying to take care of the repository in my free time (filtering and validating issues, reviewing some PR's), but I believe that we need some help for active development, do you guys have time for that? Or any suggestions?

If the suggestion is to add more members, it's ok for me, I only suggest to we invite as `maintainer` instead of `owner` the next guys.

**Comment by mohmed98** [comment]
Hellos, I want to contribute here :wave:

**Reply by danielbarion** [reply]
@aronhelser @AlexanderEllis @pdeszynski can I add @mohmed98 as `maintainer`?

**Reply by AlexanderEllis** [reply]
Personally I think that would be great, but I think @alexgurr would probably have a more authoritative opinion on this than I would :)

**Reply by alexgurr** [reply]
I think the more contributors the better, it looks like a lot of us are busy all the time! Always open to a PR to review.

**Reply by danielbarion** [reply]
Sure, I agree!

I'm trying to help in my free time but don't have time to do active development.

Let's wait @aronhelser input too

**Reply by aronhelser** [reply]
Fine with me!

**Reply by danielbarion** [reply]
Thanks guys, @mohmed98 was added as the maintainer of the project and as a member of the organization.

@mohmed98 please be kind and take a look here before any update to the project: https://github.com/ReactTooltip/react-tooltip/blob/master/contributing.md

Use `yarn cm` to do the commits and always work on branches and do merge to Master, please.

Also, every commit on the master will trigger a new release.
New release: do the flow on the file that I asked you to take a look at when you can and after that, go to GitHub releases and generate a new release from the master with the updated tag (there's a button after you select the tag as generate release notes), also you need to copy the link to changelog.md and paste like in previous releases.

Please let me know if you have any questions, thanks!

**Reply by mohmed98** [reply]
Excellent, thank you all :pray:

**Comment by danielbarion** [comment]
If anyone else wants to be an active developer in that package, please let us know!

**Comment by danielbarion** [comment]
The new thing here: everyone else wants to be an active developer in that package, please start helping solve issues with PRs and after that, you can be added as a maintainer.
